<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemTextSettings</class>
 <widget class="QWidget" name="ItemTextSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxUseRichText">
     <property name="text">
      <string>Save and display HTML and rich text</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Maximum number of lines to display (0 to show all):</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxMaxLines</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMaxLines">
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Maximum height in pixels (0 for no limit):</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxMaxHeight</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMaxHeight">
       <property name="maximum">
        <number>2000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Default style sheet:</string>
     </property>
     <property name="buddy">
      <cstring>textEditDefaultStyleSheet</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textEditDefaultStyleSheet">
     <property name="placeholderText">
      <string notr="true">a { color: blue }</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxUseRichText</tabstop>
  <tabstop>spinBoxMaxLines</tabstop>
  <tabstop>spinBoxMaxHeight</tabstop>
  <tabstop>textEditDefaultStyleSheet</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
